src/internal/git/utils.go

```diff
@@ -26,28 +28,55 @@ type Credential struct {
 	Auth http.BasicAuth
 }
 
-func MutateGitUrlsInText(host string, text string) string {
+// MutateGitURlsInText Changes the giturl hostname to use the repository Zarf is configured to use
+func MutateGitUrlsInText(host string, text string, gitUser string) string {
 	extractPathRegex := regexp.MustCompilePOSIX(`https?://[^/]+/(.*\.git)`)
 	output := extractPathRegex.ReplaceAllStringFunc(text, func(match string) string {
-		if strings.Contains(match, "/"+config.ZarfGitPushUser+"/") {
+		// TODO @JPERRY: This might be less valid of a check now that people can specify the git username
+		if strings.Contains(match, "/"+gitUser+"/") {
```

Comment:
Sorry, could you clarify a little more about what you mean by 'look for the URL instead'?

Reply:
@Racer159 I think I found a workaround for this. We only needed this check because we were using the same handler for `create` and `update` webhook triggers. 

Now, the webhook will only attempt to mutate the git URL if it's a `create`